WordPressKit/WordPressComOAuthClient.swift

```diff
@@ -78,15 +78,14 @@ public final class WordPressComOAuthClient: NSObject {
     @objc public static let WordPressComOAuthDefaultApiBaseUrl = "https://public-api.wordpress.com"
 
     enum WordPressComURL: String {
-        case oAuthBase = "/oauth2/token"
         case webauthnChallenge = "wp-login.php?action=webauthn-challenge-endpoint"
         case webauthnAuthentication = "wp-login.php?action=webauthn-authentication-endpoint"
         case socialLogin = "/wp-login.php?action=social-login-endpoint&version=1.0"
         case socialLogin2FA = "/wp-login.php?action=two-step-authentication-endpoint&version=1.0"
         case socialLoginNewSMS2FA = "/wp-login.php?action=send-sms-code-endpoint"
 
-        func url(base: String) -> URL {
-            return URL(string: self.rawValue, relativeTo: URL(string: base))!
+        func url(base: URL) -> URL {
```

Comment:
👍 for requiring `base` to be a `URL` and pushing the responsibility to create good URLs to the callers. 

IMHO, in the ideal world, we'd only have one pint in the whole app / library where we go from `String` to `URL`. The rest of the code deals with "valid" instances, without having to worry about validation.